README.md

```diff
@@ -194,7 +194,7 @@ To make things easy, you can just clone recursively:
 git clone --recursive https://github.com/PowerShell/PowerShell.git
```

Comment:
`--recursive` is not needed anymore.

Do you want to fix this or merge as is?

Reply:
I would like this to be removed as `--recursive` is not needed anymore.